multimodal/tests/unittests/others/test_deployment.py

```diff
@@ -40,9 +40,9 @@ def evaluate(predictor, df, onnx_session=None):
     "checkpoint_name",
     ["sentence-transformers/msmarco-MiniLM-L-12-v3", "sentence-transformers/all-MiniLM-L6-v2"],
 )
-@pytest.mark.skip(reason="onnx export currently requires torchtext<=0.12.0")
-def test_onnx_export(checkpoint_name):
+def test_onnx_export_hf_text(checkpoint_name):
     test_df = load_dataset("wietsedv/stsbenchmark", split="test").to_pandas()
+    test_df = test_df.head()  # subsample the data to avoid OOM in tracing
 
     predictor = MultiModalPredictor(
         problem_type="feature_extraction",
```

Comment:
Maybe consider not using this problem type and also doing predictor.fit().

Reply:
This is originally created by @FANGAreNotGnu . This PR is just trying to enable it running in the CI.

@FANGAreNotGnu any comments?

Reply:
I selected this one because it supports zero shot and at that time CI was heavy. Now we could use another task and fit(), since CI's burden is mitigated and also this task is covered by matcher.